### ISSUE TITLE
[Bug]: DocumentType enum does not include all valid options

### ISSUE THREAD

**Issue by gageorsburn** [alpacahq/alpaca-py#273]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When I call `get_account_by_id` a validation error occurs because the API is returning data that is not in the enum.

### Expected Behavior

When I call `get_account_by_id`, I expect to get back a valid `Account` object.

### SDK Version I encountered this issue in

alpaca-py version 0.81

### Steps To Reproduce

```markdown
Upload documents with other types and call the `get_account_by_id` method.
```

### Filled out the Steps to Reproduce section?

- [X] I have entered valid steps to reproduce my issue or have attached a minimally reproducible case in code that shows my issue happening; and understand that without this my issue will be flagged as invalid and closed after 30 days.

### Anything else?

<img width="673" alt="Screenshot 2023-04-18 at 12 42 14 PM" src="https://user-images.githubusercontent.com/4692734/232847454-5bd8605f-e4c5-446f-bb93-b84ce906b640.png">

**Comment by jjmaxwell4** [comment]
Also getting this error. I can't get the account Object back as it fails to parse.  When I hit `broker_client.get_account_by_id(account_id=alpacaId)` the response fails with the following:

```
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jamesmaxwell/.local/share/virtualenvs/snowballDjango-DOya9S7n/lib/python3.11/site-packages/alpaca/broker/client.py", line 192, in get_account_by_id
    return Account(**resp)
           ^^^^^^^^^^^^^^^
  File "/Users/jamesmaxwell/.local/share/virtualenvs/snowballDjango-DOya9S7n/lib/python3.11/site-packages/alpaca/broker/models/accounts.py", line 276, in __init__
    parse_obj_as(List[AccountDocument], response["documents"])
  File "pydantic/tools.py", line 38, in pydantic.tools.parse_obj_as
  File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
pydantic.error_wrappers.ValidationError: 1 validation error for ParsingModel[List[alpaca.broker.models.documents.AccountDocument]]
__root__ -> 0 -> document_type
  value is not a valid enumeration member; permitted: 'identity_verification', 'address_verification', 'date_of_birth_verification', 'tax_id_verification', 'account_approval_letter', 'limited_trading_authorization', 'w8ben' (type=type_error.enum; enum_values=[<DocumentType.IDENTITY_VERIFICATION: 'identity_verification'>, <DocumentType.ADDRESS_VERIFICATION: 'address_verification'>, <DocumentType.DATE_OF_BIRTH_VERIFICATION: 'date_of_birth_verification'>, <DocumentType.TAX_ID_VERIFICATION: 'tax_id_verification'>, <DocumentType.ACCOUNT_APPROVAL_LETTER: 'account_approval_letter'>, <DocumentType.LIMITED_TRADING_AUTHORIZATION: 'limited_trading_authorization'>, <DocumentType.W8BEN: 'w8ben'>])
  ```
  
  I created by user via the API, and it appears that a the user had a document created automatically with a NULL `document_type` field. The alpaca-py `AccountDocument` model needs to have an `Optional[ DocumentType]` field on line 33 of Documents.py
  


![image](https://github.com/alpacahq/alpaca-py/assets/1138845/81c62ecc-80da-45b0-987a-a6708a0c65e1)

**Comment by haxdds** [comment]
Should be closed by #308
